pages/studentPortal/labs/multiplication/game.tsx

```diff
@@ -3,53 +3,44 @@ import { useDispatch, useSelector } from "react-redux";
 import Winner from "../../../../components/math/longestStreak/Winner";
 import MultiplicationBlock from "../../../../components/math/longestStreak/MultiplicationBlock";
 import Rules from "../../../../components/math/longestStreak/Rules";
-import { setPlayerName, reset } from "../../../../redux/longestStreakSlice";
 
 import { Button } from "../../../../components/ui/Button";
 import {
   handlePlayerSelect,
   initializeGame,
   longestStreakSelector,
+  reset,
   setStage,
   STAGE,
 } from "../../../../redux/longestStreakSlice";
 import {
   calculatePlayerScore,
-  calculateWinner,
+  calculateWin,
   checkNumberNotSelected,
-  showWinner,
+  gameWinsMetaData,
 } from "../../../api/longestStreak";
 import { useMutation, useQuery } from "@apollo/client";
 
-import { UPSERT_GAME_LEVEL } from "../../../../graphql/longestStreak/upsertGameLevel";
-import { showEndGameImage } from "../../../api/showEndGameImage";
+import { useAuth } from "../../../../lib/authContext";
+
 import { DOWNGRADE_GAME_LEVEL } from "../../../../graphql/longestStreak/downGradeGameLevel";
 import {
   FetchGameLevelResponse,
   FETCH_GAME_LEVEL,
 } from "../../../../graphql/longestStreak/fetchGameLevel";
-import { UPDATE_GAME_LEVEL } from "../../../../graphql/longestStreak/updateGameLevel";
-import { useAuth } from "../../../../lib/authContext";
+
 import { RESET_GAME_LEVEL } from "../../../../graphql/longestStreak/resetGameLevel";
+import { UPDATE_GAME_LEVEL } from "../../../../graphql/longestStreak/updateGameLevel";
+import { UPSERT_GAME_LEVEL } from "../../../../graphql/longestStreak/upsertGameLevel";
+import UserTableStats from "../../../../components/math/longestStreak/userTableStats";
+
 export type BlockComponentGalleryProps = {
   user: any;
 };
 
 export default function BlockComponentGallery() {
```

Comment:
I would rename this function to call it LongestStreakGame and I would rename the file to be something similar as well